docs/marketplace/dev/front-end/table-design/table-features/table-feature-editable.md

```diff
@@ -0,0 +1,224 @@
+`create` - an object with config for a new rows.  
+`update` - an object with config for an existing rows.  
+`disableRowKey` - will disable row that contains mentioned column `id` (see example).  
+
+```html
+<spy-table [config]="{
+  dataSource: { ... },
+  columns: [ ... ],
+  editable: {
+    columns: [ ... ],
+    create: { ... },
+    update: { ... },
+    disableRowKey: 'col',
+  },                                                                                           
+}">
+</spy-table>
+```
+
+Let's take a closer look at all the possible options.
+
+`columns`:  
+  - `id` - a cell `id`.  
+  - `type` - a cell `type`.  
+  - `typeOptions`:  
+    - `value` - will set default value to the cell of the newly added row.  
+
+`create`:  
+  - `addButon` - an object with `title` and `icon` for the `Add button`.  
+  - `cancelButon` - an object with `title` and `icon` for the `Cancel button`.  
+  - `disableForCols` - an array with cell `ids` to be disabled.  
+  - `formInputName` - will create `input[type=hidden]` element with the specific name.  
+  - `initialData` - initials data for cells and object with errors for rows and cells.  
+  
+`update`:
+  - `url` - request url.  
+  - `saveButon` - an object with `title` and `icon` for the `Save button` (displays in the `update` popup).  
+  - `cancelButon` - an object with `title` and `icon` for the `Cancel button` (displays in the `update` popup).  
+  - `disableForCols` - an array with cell `ids` to be disabled.  `
+
+```html
+<spy-table [config]="{
+  dataSource: { ... },
+  columns: [ ... ],
+  editable: {
+    columns: [
+      { id: 'col1', type: 'edit' },
+      { id: 'col2', type: 'edit', typeOptions: { value: 'default' } },
+    ],
+    create: {
+      addButton: {
+        title: 'Add price',
+        icon: 'icon',
+      },
+      cancelButton: {
+        title: 'Cancel',
+        icon: 'icon',
+      },
+      disableForCols: ['col2'],
+      formInputName: 'form-input-name',
+      initialData: {
+        data: [
+          { col1: 'value', col2: 'value' },
+          { col1: 'value' },
+        ],
+        errors: {
+          0: {
+            rowError: 'message',
+            columnErrors: {
+              col3: 'errorMessage errorMessage errorMessage',
+            },
+          },
+        },
+      },
+    },
+    update: {
+      url: '/table-update-cell',
+      saveButton: {
+        title: 'Save',
+        icon: 'icon',
+      },
+      cancelButton: {
+        title: 'Cancel',
+        icon: 'icon',
+      },
+      disableForCols: ['col2'],
+    },
+    disableRowKey: 'col1',
+  },                                                                                           
+}">
+</spy-table>
+```
+
+## Feature Registration
+
+```ts
+@NgModule({
+  imports: [
+    TableModule.forRoot(),
+    TableModule.withFeatures({
+      editable: () => 
+        import('@spryker/table.feature.editable').then(
+          (m) => m.TableEditableFeatureModule,
+        ),   
+    }),
```

Comment:
Also show example without dynamic registration, just feature tag inside table

Reply:
added to the tech debt https://spryker.atlassian.net/browse/MP-5886